game-core/src/main/java/games/strategy/triplea/ai/pro/util/ProMatches.java

```diff
@@ -561,14 +560,14 @@ private ProMatches() {}
       final GamePlayer player, final UnitType unitType) {
     return Matches.unitIsOwnedBy(player)
         .and(Matches.unitIsOfType(unitType))
-        .and(Matches.unitIsTransporting());
+        .and(Unit::isTransporting);
   }
 
   public static Predicate<Unit> unitIsOwnedAndMatchesTypeAndNotTransporting(
       final GamePlayer player, final UnitType unitType) {
     return Matches.unitIsOwnedBy(player)
         .and(Matches.unitIsOfType(unitType))
-        .and(Matches.unitIsTransporting().negate());
+        .and(Unit::isTransporting);
```

Comment:
I think this change was unintended

Reply:
This is a good catch @RoiEXLab :+1: 
An example of me going slightly too far in simplifications.

Reply:
Fixed in: 278da46b9

Reply:
Actually fixed in: 7d7174db1